plugins/kotlin/idea/tests/testData/intentions/lambdaToAnonymousFunction/underscoreParameter.kt.after

```diff
@@ -0,0 +1,8 @@
+// IS_APPLICABLE: true
+fun foo(f: (Int, String, Int) -> String) {}
+
+fun test() {
+    foo(fun(i1: Int, s: String, i: Int): String {
```

Comment:
I think we shouldn't suggest names, so we should leave `_` as it is.
Expected result:
```kotlin
foo(fun(_: Int, _: String, i: Int): String {
    return ""
})
```